community/OM-Operator-Management/cnv-samples/cnv-operator-install/placement-cnv.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
```

Comment:
Can this be re-written to use Placement (apiVersion `cluster.open-cluster-management.io/v1beta1`)?

Reply:
So funny thing, when I created this as Placement it didn't work, because I needed to add the ManagedClusterSet binding.  I get that's how it works, but the error isn't as helpful as it could be and it will be a problem for our VMware admins.  Anyway, that's not your issue, just something I noted...